src/Core/Types/LandmarkType.cs

```diff
@@ -57,8 +58,8 @@ protected override string GetNameInProperFormat(string name, int id)
         {
             StringBuilder sb = new StringBuilder(name);
 
-            sb.Replace("UIA_", "");
-            sb.Replace("Id", "");
+            sb.Replace(DisplayStrings.UIA_, "");
+            sb.Replace(DisplayStrings.Id, "");
```

Comment:
This is mapping to a code construct. I would not recommend moving this into a resource file